packages/tsconfig-reference/copy/en/options/watchFile.md

```diff
@@ -9,4 +9,4 @@ The strategy for how individual files are watched.
 - `priorityPollingInterval`: Check every file for changes several times a second, but use heuristics to check certain types of files less frequently than others.
 - `dynamicPriorityPolling`: Use a dynamic queue where less-frequently modified files will be checked less often.
 - `useFsEvents` (the default): Attempt to use the operating system/file system's native events for file changes.
-- `useFsEventsOnParentDirectory`: Attempt to use the operating system/file system's native events to listen for changes on a file's
+- `useFsEventsOnParentDirectory`: Attempt to use the operating system/file system's native events to listen for changes on a file's parent directory
```

Comment:
This is a typo with missing phrase, just fixed (I think), @orta can you review if that's right?

Reply:
yeah